src/app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks.ts

```diff
@@ -115,3 +115,26 @@ export function useSignBitcoinNativeSegwitTx() {
     [keychain]
   );
 }
+
+// export function useCurrentAccountNativeSegwitSigner() {
```

Comment:
should it be here commented?

Reply:
its commented to avoid `unused exports` rule?

Reply:
ah yeah, meant to remove